integration_test/gce/gce_testing.go

```diff
@@ -1302,8 +1302,12 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	return vm, nil
 }
 
-func IsSUSE(imageSpec string) bool {
-	return strings.HasPrefix(imageSpec, "suse-") || strings.HasPrefix(imageSpec, "opensuse-") || strings.Contains(imageSpec, "sles-")
+func IsSLES(vm_OS_ID string) bool {
```

Comment:
Should these functions operate on `vm` or `vm.OS`?

Reply:
I think we should pass `vm`, it seems the most future-proof.

Reply:
I created `IsSUSEVM(vm *VM) bool` that operates directly on `vm`.